mpp/include/mpp_domains_util.inc

```diff
@@ -287,7 +287,11 @@
   end subroutine mpp_set_super_grid_indices
 
   !> @brief Modifies the indices of the input domain to create the supergrid domain
-  subroutine mpp_create_super_grid(domain)
+  !> @example This is an example of how to use mpp_create_super_grid_domain
+  !! call mpp_copy_domain(domain_in, domain_out)
+  !! call super_grid_domain(domain_out)
+  !! domain_in is the original domain, domain_out is the domain with the supergrid indices
```

Comment:
@bensonr This addresses your comment *Since this is a destructive operation, there should be doxygen information on how to utilize this function (copy, modify).*

Reply:
Thanks.  I should have been more specific about this information also being in mpp_domains.F90.  I use that as my first place for all documentation on usage, etc.  I'll approve and we can augment at a later date.